### DISCUSSION TITLE
iMessage icon

### DISCUSSION THREAD

**Discussion by SingularDave** [thuliteio/doks#791] [Q&A] [answered]
Hello all. This might be a stupid question.  Which images and site description are used for the site when you send the link as a iMessage? I changed the favicons in the static file—except the svg ones— to my logo, but I’m still getting doc icon when I text a link to someone. Are the svg images being used? If so, does anyone know the best way to convert your png logo to svg? Thanks.

**Comment by roheclassic** [comment, marked as answer]
I think it can be specified in param.toml.

```
## Open Graph
images = ["XXXXXXX.png"] <---*
ogLocale = "en_US"
domainTLD = "doks.netlify.app"
titleHome = "Doks Theme"
```

In addition, if you specify  ```images``` in each content file (index.md), the image will be shared preferentially for that content page.

**Reply by SingularDave** [reply]
Thanks for the reply. That change did not seem to make a difference. I've replaced all of the logo images I can find in the "node_modules/@hyas/doks/static" file. Yet I'm still getting the paper image icon on my browser and with iMessage it says "Modern Documentation Theme" with the same paper image icon. I just can't seem to find where it's located in the files. Thanks again for the help.

**Reply by roheclassic** [reply]
- What is the value below? (These are displayed as text, image for the top page.)
	- ```title``` and ```description``` in ```content/en/_index.md```
	- ```images``` in ```content/en/_index.md```
- Also, does your own site is multilingual site? (If so, ```static``` directory is required for each language.)
- Please the information here as shown by ```npm run check```.
- Do you have a repo you can share for this problem?

**Comment by SingularDave** [comment]
Problem solved @roheclassic. It was title and description in content/en/_index.md. Thank you!
